jujugui/static/gui/src/app/components/deployment-flow/deployment-flow.js

```diff
@@ -102,7 +102,12 @@ class DeploymentFlow extends React.Component {
         });
         return;
       }
-      this.setState({ddEntity: this.props.makeEntityModel(data[0])});
+      const entityModel = this.props.makeEntityModel(data[0]);
+      this.props.displayPostDeployment(
```

Comment:
Are we displaying post deployment before the deployment here?
This seems confusing.